### PR TITLE
Task_1 solution file

### PR DESCRIPTION
**Pull request by kavi-12**
__1__ 

#### Short description of what this resolves:
python script with reference to the original csv file

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by Lokesh2703** [comment]
There is no file here.

**Comment by kavi-12** [comment]
file is in Submissions/kavi-12/task-1/csvtojson.py
in the "newbranch" (may be by changing the branch)

File: Submissions/kavi-12/#task-1/csvtojson.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+data = {}
+with open(csv_path,  encoding='utf-8') as csvfile:
+    csvReader = csv.DictReader(csvfile)
+    for csvrow in csvReader:
+        
+        obj = {
+            "title": csvrow["title"],
+            "genres": csvrow["genres"]
+
```
Comment (deepakDOTexe) [inline Submissions/kavi-12/#task-1/csvtojson.py:18]: In the json file created movieId is missing so make changes in your code accordingly.

**Review by kavi-12 — COMMENTED** [review]
Movie id is the key with the values as the object title and genres

**Comment by deepakDOTexe** [comment]
You should have put movieId inside the object as the values have have three attributes.

**Comment by kavi-12** [comment]
movieId added

**Pull request merged**